extensions/wikia/Wall/WallHistory.class.php

```diff
@@ -156,7 +156,8 @@ private function internalAdd( $parentPageId, $postUserId, $postUserName, $isRepl
 				'parent_page_id' => $parentPageId,
 				'post_user_id' => $postUserId,
 				'post_ns' => MWNamespace::getSubject( $ns ),
-				'post_user_ip' => ( intval( $postUserId ) === 0 ? $this->ip2long( $postUserName ) : null ),
+				'post_user_ip' => null, // TODO SUS-2257 Mix <mix@fandom.com> Remove after migration.
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)